sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md

```diff
@@ -476,4 +476,8 @@ directive:
     where: $.definitions.RecoveryPointProperties.properties.expiryTime
     transform: >
       $["format"] = "date-time";
+  - from: bms.json
```

Comment:
Add a TODO here to remove this workaround once swagger got fixed.

Reply:
Fixed